server/seed/seedCohorts.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-console */
+
+const Cohort = require('../models/cohort.model');
+require('dotenv').config();
+
+const userId = process.env.USER_ID;
+const initialCohorts = [
+  {
+    _id: '5c9a12eb46e555ba24992a2c',
+    adminIds: [userId],
```

Comment:
Only this one property is constant (userId). Could we generate properties automatically? (e.g. generate cohort id - random 24 chars string) and other properties via some function or loop. And after that would get cohort from db and add list for them in the same way. I mentioned it because when we change models etc. we would have to edit whole jsons. If data would be generated automatically there will be one place to change it.